test/application/team-validator.js

```diff
@@ -34,6 +34,22 @@ describe('Team Validator features', function () {
 			});
 		});
 
+		it('should reject non-existent natures', function (done) {
+			let packedTeam = "|pikachu|eviolite||thunderbolt|nonexistentNature||||||";
+			TeamValidator('customgame').prepTeam(packedTeam).then(result => {
```

Comment:
idk why people use async code in tests.

You could just use `validateTeam` instead of `prepTeam` and then everything would be synchronous.